docs/architecture.md

```diff
@@ -1,33 +1,67 @@
-# MiniMint Architecture
+# Minimint Architecture
 
-MiniMint started out as a federated Chaumian e-cash prototype. By now it is a more general framework for federated financial applications. At its core lies the ability to agree on and process transactions. The possible input and output types of these transactions are defined by modules.
+Minimint is a general framework for building federated financial applications.
+The current implementation allows users to make private low-fee payments through a federated [Chaumian e-cash](https://en.wikipedia.org/wiki/Ecash) mint that processes deposits and withdrawals over Bitcoin or Lightning.
```

Comment:
```suggestion
The current implementation allows users to make private low-fee payments through a federated [e-cash](https://en.wikipedia.org/wiki/Ecash) mint that processes deposits and withdrawals over Bitcoin or Lightning.
```
Since there are no chaumian signatures used... But I get that this is a recognizable term, probably alright.

Reply:
I think Chaumian e-cash is kinda ok since we use a blind signature scheme, just not Chaum's RSA based one.

Reply:
True. But note that even the Wikipedia article is titled eCash...

Reply:
How about "federated blinded mint that issues e-cash"?